style.css

```diff
@@ -0,0 +1,3 @@
+.header{
+color:blue;
+}
```

Comment:
Please ensure that the styling as recommended for the `h1` tag is implemented on the right class since there is no `.header` class in your HTML file. :smile